shell/platform/windows/client_wrapper/flutter_window_controller.cc

```diff
@@ -12,33 +12,23 @@ namespace flutter {
 FlutterWindowController::FlutterWindowController(
```

Comment:
Why does this file still exist? I would expect the change to a view controller to be a move, not a copy.

Reply:
Fixed in next commit